### PR TITLE
[Fix #114291643] Refactor the project to a single master page and delete the other two master pages

### PR DESCRIPTION
**Pull request by ademola-raimi**
I refactored the project by making sure that all the pages extending from different master pages are now extending from a single master page. I also deleted all unused pages  This is to ensure that we do not violate the DRY principle.

### CODE REVIEW

File: resources/views/app/includes/contents/favorites.blade.php [hunk 1]
```diff
@@ -138,7 +139,7 @@
         <!-- end card -->
 
         @empty
-            <p>No Episodes to display</p>
+            <p>Favourite episodes not available </p>
```
Comment (johnkariuki) [inline resources/views/app/includes/contents/favorites.blade.php:142]: _Code functionality - Critical_

@andela-araimi 

Write tests for this to assert that when a user clicks on favorite episodes, they get the 'Favorite episodes not available'.

Also, Consider rephrasing the message. `Not available` may refer to an error, not zero favorites.

Comment (andela-jkariuki) [inline resources/views/app/includes/contents/favorites.blade.php]: *Code functionality - Critical*

@andela-araimi 

Write tests for this to assert that when a user clicks on favorite episodes, they get the 'Favorite episodes not available'.

Also, Consider rephrasing the message. `Not available` may refer to an error, not zero favorites.

File: resources/views/app/master.blade.php [hunk 2]
```diff
@@ -44,11 +45,16 @@
         <link href = "{!! load_asset('/library/sweetalert/sweetalert.css') !!}" rel="stylesheet"  />
         <script type="text/javascript" src = "{!! load_asset('/library/sweetalert/sweetalert.min.js') !!}"></script>
 
+        <script type="text/javascript" src = "{!! asset('/js/LoginAndSignup.js') !!}"></script>
+        <script type="text/javascript" src = "{!! asset('/js/PasswordReset.js') !!}"></script>
+        <script type="text/javascript" src = "{!! asset('/js/NewPassword.js') !!}"></script>
```
Comment (johnkariuki) [inline resources/views/app/master.blade.php]: **Code Functionality - Nitpick**

Please replace asset with load_asset to ensure it sets HTTPS on production environment for referencing the scripts.

File: app/Http/Controllers/EpisodeController.php [hunk 3]
```diff
@@ -41,8 +41,9 @@ public function index()
     public function allEpisode()
     {
         $episodes = Episode::get();
+	    $channels = Channel::all();
```
Comment (unicodeveloper) [inline app/Http/Controllers/EpisodeController.php:44]: **Code  Quality - Nitpick**

Please convert your tabs to spaces in your IDE & use 4 spaces for 1 tab

File: resources/views/app/includes/contents/favorites.blade.php (hunk 1 above)
Comment (unicodeveloper) [inline resources/views/app/includes/contents/favorites.blade.php:142]: :+1:  on writing tests

**Pull request merged**